test/e2e/tests/eth-sign.spec.js

```diff
@@ -31,10 +59,18 @@ describe('Eth sign', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
 
+        // Enable eth_sign
```

Comment:
Instead of enabling `eth_sign` with Settings, we could skip all the steps below by enabling it from the state. This will make the test faster and complies with the standards followed on e2e.

Detailed explanation: on `fixture-builder` file, we could add the state for `eth_sign` and set it to `true` on a function below, in the same way it's done with other preferences.

Then on this test, we can just enable that fixture like this example:

```
.withPreferencesController({
            preferences: {
              showFiatInTestnets: true,
            },
```

Reply:
Thanks @seaona - changed in https://github.com/MetaMask/metamask-extension/pull/17308/commits/c51da5e223c179e096ddd96fced8416bd827d486